sources/packages/web/src/services/UserService.ts

```diff
@@ -18,14 +18,8 @@ export class UserService {
     return ApiClient.User.checkUser(authHeader);
   }
 
-  async getBCeIDAccountDetails(
-    authHeader?: any,
-  ): Promise<BCeIDDetailsDto | null> {
-    try {
-      return await ApiClient.User.bceidAccount(authHeader);
-    } catch (excp) {
-      return null;
-    }
+  async getBCeIDAccountDetails(authHeader?: any): Promise<BCeIDDetailsDto> {
```

Comment:
Removing the option to this method no longer returns null will affect the below code?
![image](https://user-images.githubusercontent.com/61259237/132389671-bbc4b791-9d5c-46e9-a7cd-c1f0823319d6.png)